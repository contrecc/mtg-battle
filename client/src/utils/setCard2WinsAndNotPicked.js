export default function setCard2WinsAndNotPicked(pickedCard) {
  return function(state, props) {
    return {
      winner: false,
      winStreak: 0,
      winningCard: 'card2',
      roundCompleted: true,
      pickedCard: pickedCard
    };
  };
}
