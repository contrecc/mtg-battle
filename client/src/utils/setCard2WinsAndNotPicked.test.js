import setCard2WinsAndNotPicked from './setCard2WinsAndNotPicked';

describe('setCard2WinsAndNotPicked', () => {
  it('should return a function', () => {
    expect(setCard2WinsAndNotPicked()).toMatchSnapshot();
  });

  it('should return a function that then updates the state', () => {
    const returnedFunction = setCard2WinsAndNotPicked();
    expect(returnedFunction({}, {})).toMatchSnapshot();
  });
});
