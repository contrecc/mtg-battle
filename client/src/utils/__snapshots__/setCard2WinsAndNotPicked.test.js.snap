// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`setCard2WinsAndNotPicked should return a function 1`] = `[Function]`;

exports[`setCard2WinsAndNotPicked should return a function that then updates the state 1`] = `
Object {
  "roundCompleted": true,
  "winStreak": 0,
  "winner": false,
  "winningCard": "card2",
}
`;
